Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -112,6 +122,7 @@
                 pt=(150 * u.MeV, None),
                 removeNeutral=True,
             ),
+            # outputDirRoot=outputDir,
             outputDirCsv=outputDir,
```

Comment:
```suggestion
            outputDirRoot=outputDir,
            # outputDirCsv=outputDir,
```

if you want to include this fix here. otherwise I can do it in a followup pr

Reply:
So I didn't change this because I am not sure it won't break something in the test or not. If you don't mind doing it as a follow up it would speed up the process of getting this merged

Reply:
I think this is a debugging leftover from someone

sure no worries